opensearch-operator/controllers/suite_test_helpers.go

```diff
@@ -95,11 +95,10 @@ func ArrayElementContains(array []string, content string) bool {
 }
 
 func ComposeOpensearchCrd(clusterName string, namespace string) opsterv1.OpenSearchCluster {
-
 	OpensearchCluster := &opsterv1.OpenSearchCluster{
 		TypeMeta: metav1.TypeMeta{
 			Kind:       "OpenSearchCluster",
-			APIVersion: "opensearch.opster.io/v1",
+			APIVersion: "github.com/Opster/opensearch-k8s-operator/opensearch-operator/v1",
```

Comment:
Must stay `opensearch.opster.io` as this is the kubernetes apiversion and not the go package.

Reply:
true, missed that one, updated the PR.